src/Context/MetaContext.php

```diff
@@ -271,4 +271,105 @@ public function thePageMetaDescriptionShouldNotExist(): void
             'Meta description does exist'
         );
     }
+
+    /**
+     * @Then the page meta robots should be nofollow
+     */
+    public function thePageShouldBeNofollow(): void
+    {
+        $metaRobotsElement = $this->getMetaRobotsElement();
+
+        Assert::notNull(
+            $metaRobotsElement,
+            'Meta robots does not exist.'
+        );
+
+        Assert::contains(
+            strtolower($metaRobotsElement->getAttribute('content') ?? ''),
+            'nofollow',
+            sprintf(
+                'Url %s is not nofollow: %s',
+                $this->getCurrentUrl(),
+                $metaRobotsElement->getHtml()
+            )
+        );
+    }
+
+    /**
+     * @Then the page meta robots should not be nofollow
+     */
+    public function thePageShouldNotBeNofollow(): void
+    {
+        $this->assertInverse(
+            [$this, 'thePageShouldBeNofollow'],
+            'Page meta robots is nofollow.'
+        );
+    }
+
+    private function getImageElement(): ?NodeElement
+    {
+        return $this->getSession()->getPage()->find('css', 'img');
+    }
+
+    /**
+     * @Then the images should have alt text
+     */
+    public function theImagesShouldHaveAltText(): void
+    {
+
+        $imageElements = $this->getImageElement();
+
+        foreach($imageElements as $imageElement)
+        {
+          Assert::notNull($imageElement);
+          $imageAlt = $imageElement->getAttribute('alt');
+          Assert::notEmpty($imageAlt,'Alt Text is empty for image: ' + $imageElement);
+        }
+    }
+
+    private function getViewportElement(): ?NodeElement
+    {
+        return $this->getSession()->getPage()->find(
+            'xpath',
+            '//head/meta[@name="viewport"]'
+          );
+    }
+
+    /**
+     * @Then the site should be responsive
+     */
+    public function theSiteShouldBeResponsive(): void
```

Comment:
It would fit better in a new UXContext. If we create that new context, we will be able to make it more precise (in the future) by checking more things than the viewport without polluting the MetaContext.